openstf/pipeline/predict_sklearn.py

```diff
@@ -1,18 +1,19 @@
-from datetime import datetime, timedelta
+from datetime import datetime, timedelta, timezone
 from pathlib import Path
 
 import joblib
 import pandas as pd
-
+import structlog
```

Comment:
I would suggest to add structlog to the `requirements.txt` since we are using it now.

Reply:
(`requirements.txt` lists all the dependencies of a project in Python)

Reply:
I added structlog 21.1.0 to the requirements.

Reply:
perfect!